diff -up highlight-3.42/src/core/platform_fs.cpp.list-available-langs highlight-3.42/src/core/platform_fs.cpp
--- highlight-3.42/src/core/platform_fs.cpp.list-available-langs	2019-01-11 11:42:30.948013206 +0100
+++ highlight-3.42/src/core/platform_fs.cpp	2019-01-11 11:44:25.206011626 +0100
@@ -222,8 +222,10 @@ void getFileNames ( const string &direct
     if ( errno ) return;
 
     // sort the current entries for fileName
-    if ( firstEntry < fileName.size() )
-        sort ( &fileName[firstEntry], &fileName[fileName.size() ] );
+    // https://gitlab.com/saalen/highlight/issues/84
+    // coredump with GLIBCXX_ASSERTIONS compiler option, use iterators if sort is needed
+    //if ( firstEntry < fileName.size() )
+    //    sort ( &fileName[firstEntry], &fileName[fileName.size() ] );
 
     // recurse into sub directories
     // if not doing recursive, subDirectory is empty
